frontend/.eslintrc.cjs

```diff
@@ -25,6 +25,8 @@ module.exports = {
         'import/no-undef': 'off',
         'object-curly-newline': 'off',
         'no-console': 'off',
+        'react/require-default-props': 'off',
+        'no-nested-ternary': 'off',
```

Comment:
Thanks, default props was annoying the hell out of me. I found a solution for it I could share and we could discuss the importance of this. As long as we handle default values properly I think we could avoid need for this rule. Let's see how this goes.

Reply:
Absolutely!